templates/web/base/dashboard/index.html

```diff
@@ -152,31 +74,85 @@ <h2>[% loc('Reports') %]</h2>
         <label for="end_date">[% loc('End Date') %]</label>
         <input name="end_date" type="date" value="[% end_date | html %]" class="form-control">
     </p>
-    <p class="no-label">
-        <input type="submit" class="btn" value="[% loc('Look up') %]">
-        <a class="btn export_as_csv" href="[% c.req.uri_with({ export => 1 }) %]">[% loc('Export as CSV') %]</a>
-    </p>
 </div>
 
-<table width="100%" id="reports">
+<p align="center">
+    <input type="hidden" name="body" value="[% body.id | html %]">
+    <input type="submit" class="btn" value="[% loc('Look up') %]">
+    <input type="submit" class="btn" name="export" value="[% loc('Export as CSV') %]">
+</p>
```

Comment:
Might be worth making the grouping a hidden field here otherwise if you drill down it resets the grouping to the default.

Reply:
Ooh, yes, will do.